src/compiler/worker/initWorker.js

```diff
@@ -34,6 +34,8 @@ module.exports = function initWorker({
     );
   }
 
+  const outputPath = JSON.parse(options).configOptions.assetsDest;
```

Comment:
it doesn't seem that you need to use `json.parse` here
```suggestion
  const outputPath = options.configOptions.assetsDest;
```

Reply:
From what I saw and experienced, this worker has its params injected as JSON, since it's a forked sub-process. They come from [here](https://github.com/callstack/haul/blob/next/src/compiler/createForkProcess.js#L38) and are also read [here](https://github.com/callstack/haul/blob/next/src/compiler/worker/runWebpackCompiler.js#L25).

Reply:
I misread this type definition, sorry!